immunopepper/namedtuples.py

```diff
@@ -15,16 +15,13 @@
 """
 Output_junc_peptide namedtuple
 - output_id: (gene_id).(junction_id). gene_id is the index of given gene in the splicegraph array.
-- junction_id is the index of given junction pair in all junction pair (in descending or ascending order)
 - peptide: (peptide_string). The peptide translated from junction pairs.
 - exons_coor: Coord namedtuple
-- junction_annotated: bool. True if the junction also appears in the input annotation file, False otherwise. 
-  False if the peptide stops before the junction  
+- strand: str ('+', '_'). The strand of gene.
 - read_frame_annotated: bool. True if a transcript in the given reading frame is present in the annotation, 
   False if it is created by propagation
 """
-OutputPeptide = namedtuple('OutputPeptide', ['output_id', 'peptide', 'exons_coor', 'junction_expr',
-                                                     'junction_annotated', 'read_frame_annotated'])
+OutputPeptide = namedtuple('OutputPeptide', ['output_id', 'peptide', 'exons_coor', 'strand', 'read_frame_annotated'])
```

Comment:
I have two output file 
(1) Peptide metadatafile 
(2) Kmer file 
I used to have both junction_annotated and read_frame annotated flags in both files. However I removed the junction_annotated from the peptide metadata file (1) because I felt that it was sufficient to have the information at the kmer level. 
The kmer can either be 
- Not crossing a junction --> No junction_annotated flag
- Crossing 1 junction -->  junction_1_annotated flag
- Crossing 2 junctions --> junction_1 or junction_2 is annotated
See `[here](https://github.com/ratschlab/immunopepper/blob/78d2f92bab5a51e9d2260339c62766de553b1678/immunopepper/traversal.py#L621)` 
I could add the junction_annotated flag to the peptide metadata file (1) if I extract the junction coordinates according to the strand and perform the lookup in gene_annot_jx

Reply:
No, I think this is good then. Wasn't clear from the context that the information is output elsewhere.